paper.md

```diff
@@ -59,11 +59,11 @@ The advantages not withstanding, a major impediment to the application of SEMs h
 In 2002 Sherwin and Peiro [@Sherwin:2002vx] wrote:  "The development of robust unstructured high-order methods is currently limited by the inability to consistently generate valid computational meshes for complex geometries without user intervention." This has remained true particularly for quadrilateral and hexahedral meshes.
 For these reasons, HOHQMesh was developed to generate all-quadrilateral and extruded hexahedral meshes suitable for use with spectral element methods. HOHQMesh is a direct quadrilateral mesher, which generates quadrilateral elements by the subdivision method of Schneiders [@schneiders2000algorithms] rather than indirectly from a triangular mesh or by curving a low order mesh. It also adjusts the size and curvature of the elements based on the length scales in the model, rather than attempting to modify an existing low-order mesh.
 
-Examples of meshes generated by HOHQMesh have been published in [@winters2014high], [@kopriva2016geometry], [@acosta2020simulation], [@manzanero2020entropy], [@ersing2024entropy], [@ranocha2024robustness], [@PhysRevFluids.9.053901], plus [@wintermeyer2018] and [@eriksson2024laplacian].
+Examples of meshes generated by HOHQMesh have been published in @winters2014high, @kopriva2016geometry, @acosta2020simulation, @manzanero2020entropy, @ersing2024entropy, @ranocha2024robustness, @PhysRevFluids.9.053901, plus @wintermeyer2018 and @eriksson2024laplacian.
 
 # Features
 
-HOHQMesh is designed to require minimal input from the user through the use of a control file. The model defines the geometry in terms of an outer and inner boundary curves.
+HOHQMesh is designed to require minimal input from the user through the use of a control file. The model defines the geometry in terms of outer and inner boundary curves.
```

Comment:
@DavidAKopriva @andrewwinters5000 I concur with the editor that sentence is currently at least ambiguous. Is fix proposed in the PR the correct one? Or should it be something like the following?
```suggestion
HOHQMesh is designed to require minimal input from the user through the use of a control file. The model defines the geometry in terms of an outer and one or more inner boundary curves.
```